docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
 
   orthanc:
     image: brianzhouzc/orthanc-tmi-plugins:latest
-    entrypoint: ['Orthanc']#, '--verbose']
+    entrypoint: ['Orthanc', '--verbose']
```

Comment:
Probably should comment out verbose

Reply:
I enabled it because `DEBUG_LOG` can disable verbosity. I think it is currently disabled, would need to check `<core.h>`.